backend/src/test/resources/queries/facility-create

```diff
@@ -22,5 +22,7 @@ mutation addFac($deviceId: String!, $deviceSpecimenTypes: [ID]!) {
     deviceTypes: [$deviceId]
     deviceSpecimenTypes: $deviceSpecimenTypes
     defaultDevice: $deviceId
-  )
+  ) {
+    id
```

Comment:
If we're returning a subfield in this query we will need to use the new `addFacilityV2` mutation to avoid errors.